build/gulpfile.reh.js

```diff
@@ -396,12 +396,55 @@ function tweakProductForServerWeb(product) {
 			));
 			gulp.task(serverTaskCI);
 
+			const createDummyGitHubAuthExtensionTask = task.define('createDummyGitHubAuthExtensionTask', (done) => {
+				const dir = path.join(BUILD_ROOT, destinationFolderName, 'extensions', 'github-authentication');
```

Comment:
Add a comment here explaining why we need this dummy extension hack :smiling_face_with_tear:

Reply:
good idea @jeanp413 👍 

commented added, would you change anything in it? Is it clear enough?